test/unit/api_spec.js

```diff
@@ -4385,7 +4381,8 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       const viewport = page.getViewport({ scale: 1 });
       expect(viewport instanceof PageViewport).toEqual(true);
 
-      const canvasAndCtx = CanvasFactory.create(
+      const { canvasFactory } = pdfDocument;
+      const canvasAndCtx = canvasFactory.create(
```

Comment:
In the other tests we call `canvasFactory.destroy(canvasAndCtx);` at the end, but we don't seem to do that here. Should we do that? (If so, it's probably a pre-existing issue.)

Reply:
> Should we do that? (If so, it's probably a pre-existing issue.)

Yes, it may be a small "problem" however it's obviously the correct thing to do. (And yes, it's definitely a pre-existing issue.)